lightning/src/ln/chan_utils.rs

```diff
@@ -479,31 +492,31 @@ pub(crate) fn get_htlc_redeemscript_with_explicit_keys(htlc: &HTLCOutputInCommit
 	}
 }
 
-/// note here that 'a_revocation_key' is generated using b_revocation_basepoint and a's
+/// note here that 'broadcaster_revocation_key' is generated using countersignatory_revocation_basepoint and broadcaster's
```

Comment:
Note your fault but this comment is just stale - Probably replace it with "Gets the witness redeemscript for an HTLC output in a commitment transaction. Note that `htlc` does *not* need to have its previous_output_index filled."

Reply:
Added a commit.